client/src/types/event-data.ts

```diff
@@ -37,6 +37,9 @@ export interface PlayXSoundEvent {
 export interface SayEvent {
   id: 'SayEvent';
   text: string;
+  audio: {
```

Comment:
shouldn't this be nullable?
Do we want to preserve fallback to browser tts if server side fails?

Reply:
This is the same problem I've mention in the other comment - I don't know how do we want to handle speech synthesis error. I really don't want to reimplement browser based speech synthesis because it's buggy and unreliable and works differently across browsers and the implementation sucks. So right now it's either with audio or no speech at all.